tests/test_comments.py

```diff
@@ -102,3 +104,91 @@ def test_empty_issue(self):
+                                "created_on": (datetime.now() - timedelta(days=params[3])).strftime('%Y-%m-%dT%H:%M:%SZ')
+                            },
+                        ],
+                    },
+                }
+
+                backlogger.json_rest = MagicMock(return_value=rest)
+                backlogger.issue_reminder(
+                    {"query": "query_id=123&c%5B%5D=updated_on"},
+                    {"priority": {"name": params[0]}, "id": 1000},
+                )
+                assert backlogger.has_repeat_reminder == True
+                backlogger.json_rest.assert_any_call(
+                    "GET",
+                    "https://example.com/wiki/1000.json?include=journals",
+                )
+
+                out, err = self.capsys.readouterr()
+                assert re.search("Automatic degrade priority from {} to {} for 1000".format(params[0],params[1]), out)
+
+
+    def test_issue_with_low_priority_never_change(self):
+        test_params = [
+            # slo_from, slo_id, slo_days
+            ["Low", 3, 2],
+            ["Low", 3, 1000]]
+        for params in test_params:
+            with self.subTest(params):
+                data = {"url": "https://example.com/issues", "web": "https://example.com/wiki",
+                        "api": "https://example.com/issues.json",
+                        "reminder-comment-on-issues": True}
+                backlogger.data = data
+                rest = {
+                    "issue": {
+                        "id": 1000,
+                        "priority": { "id": params[1], "name": params[0] },
+                        "journals": [
+                            { "id": 1, "notes": "" },
+                            { "id": 2, "notes": None },
+                            {
+                                "id": 3,
+                                "notes": "This ticket was set to **Urgent** priority but was not updated [within the SLO period](https://example.com/issues). Please consider picking up this ticket or just set the ticket to the next lower priority.",
+                                #"created_on": (datetime.now() - timedelta(weeks=3)).strftime('%Y-%m-%dT%H:%M:%SZ')
+                                "created_on": (datetime.now() - timedelta(days=params[2])).strftime('%Y-%m-%dT%H:%M:%SZ')
+                            },
+                        ],
+                    },
+                }
+
+                backlogger.json_rest = MagicMock(return_value=rest)
+                backlogger.issue_reminder(
+                    {"query": "query_id=123&c%5B%5D=updated_on"},
+                    {"priority": {"name": params[0]}, "id": 1000},
+                )
+                assert backlogger.has_repeat_reminder == True
+                backlogger.json_rest.assert_any_call(
+                    "GET",
+                    "https://example.com/wiki/1000.json?include=journals",
+                )
+
+                out, err = self.capsys.readouterr()
```

Comment:
Please extract the mocking to reduce the duplication within the two test functions